### DISCUSSION TITLE
Allow installation from private repositories

### DISCUSSION THREAD

**Discussion by mhughes2k** [espanso/espanso#1182] [Feature requests and Ideas]
Just found this tool and love the fact that it can install from a repo, but we were looking at setting up a package that is subject-domain constrained and was wondering if it was feasible to install not just from public repos but also from private ones (providing appropriate authentication could be provided).

**Comment by federico-terzi** [comment]
Hey 

Installing from private repositories is currently not possible, but I'll make sure to add it to the roadmap.
In the meanwhile, you could use a combination of symlinks similar to what is described into the [sync docs](https://espanso.org/docs/sync/). For example, you could create a single config file with all your private matches and share it with your colleagues, ecc.

Cheers :)

**Comment by smeech** [comment]
https://espanso.org/docs/packages/external-packages/#private-alternatives
